dash_app/dashboard.py

```diff
@@ -55,7 +62,62 @@ def _generate_next_value_(name, start, count, last_values):  # pylint: disable=n
     REGION_GRAPH = enum.auto()
     REGION_TAG_TABLE = enum.auto()
     DATATABLE_REGIONS = enum.auto()
-    # TODO(tom): Add remained of dash ids to this enum.
+    DATASETS_DROPDOWN = enum.auto()
+    REGIONS_VARIABLE_DROPDOWN = enum.auto()
+    PIVOT_TABLE_PRESETS_DROPDOWN = enum.auto()
+
+
+@enum.unique
+class DashboardFile(GetByValueMixin, str, enum.Enum):
+    """Enum of files that may be displayed in the dashboard."""
+
+    # Define a custom __new__ so DashboardFile instances use 'filename' as their value and have a
+    # description.
+    def __new__(cls, filename, description):
+        # Initialize super class (str) with filename.
+        obj = super().__new__(cls, filename)
+        # _value_ is a special name of enum. Set it here so enum code doesn't attempt to call
+        # str(filename, description).
+        obj._value_ = filename
+        obj.description = description
+        obj.filename = filename
```

Comment:
nit: FWIW- It was confusing to me why we have a `filename` here but then instead of using it, code goes and looks up the corresponding `dataset_utils.MANUAL_FILTER_REMOVED_WIDE_DATES_CSV_PATH` constants...  I guess this `filename` is really just a unique ID used in the html.  It might make sense to rename it if we aren't planning to actually use it as a filename.